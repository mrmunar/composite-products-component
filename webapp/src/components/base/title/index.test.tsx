import React from 'react';
import { render, cleanup, fireEvent, screen } from '@testing-library/react';
import Title from './';

afterEach(cleanup);

test('should take a snapshot', () => {
    const { baseElement } = render(<Title />);
    expect(baseElement).toMatchSnapshot();
});

test('can have a value', () => {
    const { getByTestId } = render(<Title>Test Title</Title>);
    expect(getByTestId('title')).toHaveTextContent('Test Title');
});