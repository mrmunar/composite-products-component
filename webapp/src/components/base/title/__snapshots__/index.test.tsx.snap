// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<body>
  <div>
    <h3
      data-testid="title"
    />
  </div>
</body>
`;
